docker-compose.override.yml

```diff
@@ -1,19 +1,18 @@
-version: "3"
+version: "3.4"
 
 services:
   freshlytics:
+    image: freshlytics/freshlytics:development
     build:
-      context: .
-      dockerfile: Dockerfile.dev
+      target: development
     ports:
       - "3000:3000"
       - "3001:3001"
     volumes:
-      - ./src/client:/app/src/client
-      - ./src/server:/app/src/server
+      - ./src/client:/usr/src/freshlytics/client:rw
+      - ./src/server:/usr/src/freshlytics/server:rw
     environment:
```

Comment:
Does it make sense to remove this as well as it's defined in the Dockerfile?

Reply:
Sorry, just for clarification, are you referring to the volume mounting bit?
If you are, you would want to leave it in so you can hot-reload your changes during development.

Let me know if you meant something else!

Reply:
Sorry for not being clear, I meant the `NODE_ENV=development` part.